### PR TITLE
Fix errors found when comparing podman v1 --help versus V2

### PR DESCRIPTION
**Pull request by rhatdan**
Mainly add missing commands to podman image, podman containers, podman system

Also fix some informations messages and descriptions.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @baude @vrothberg @giuseppe @TomSweeneyRedHat @QiWang19 @jwhonce PTAL

File: cmd/podman/containers/prune.go [hunk 1]
```diff
@@ -18,10 +18,10 @@ import (
 var (
 	pruneDescription = fmt.Sprintf(`podman container prune
 
-	Removes all stopped | exited containers`)
+	Removes all not running containers`)
```
Comment (mheon) [inline cmd/podman/containers/prune.go:21]: Should this be "non running"?

Comment (rhatdan) [reply cmd/podman/containers/prune.go:21]: Yup

**Comment by QiWang19** [comment]
LGTM

File: cmd/podman/containers/prune.go (hunk 1 above)
Comment (rhatdan) [reply cmd/podman/containers/prune.go]: Fixed

**Comment by baude** [comment]
all aliased commands need to have a completely separate command for them to so that example text is contextually correct.  the team outvoted me on that, so this is not a baude edict.  you can look at some of the image or container commands as to how it should be done.  in those cases, the flags have to be broken out and into a func that handles a pflag.FlagSet ...

**Comment by rhatdan** [comment]
@baude I see the correct help, I am not sure what command you are talking about?
```./bin/podman image import --help
Import a tarball to create a filesystem image

Description:
  Create a container image from the contents of the specified tarball (.tar, .tar.gz, .tgz, .bzip, .tar.xz, .txz).

  Note remote tar balls can be specified, via web address.
  Optionally tag the image. You can specify the instructions using the --change option.

Usage:
  podman image import [flags] PATH [REFERENCE]

Examples:
  podman image import http://example.com/ctr.tar url-image
  cat ctr.tar | podman -q image import --message "importing the ctr.tar tarball" - image-imported
  cat ctr.tar | podman image import -

Flags:
  -c, --change stringArray   Apply the following possible instructions to the created image (default []): CMD | ENTRYPOINT | ENV | EXPOSE | LABEL | STOPSIGNAL | USER | VOLUME | WORKDIR
  -m, --message string       Set commit message for imported image
  -q, --quiet                Suppress output
```

**Comment by edsantiago** [comment]
@rhatdan I don't see any inconsistencies in your addition; the only broken one right now is `podman unpause`, which is not something you touch in this PR. Your PR LGTM.

**Pull request merged**